File: swagger/models/get_sites_response.py
from .base_model import BaseModel

from .pagination_response import PaginationResponse
from .site import Site

class GetSitesResponse(BaseModel):
	"""
	no description available

	Attributes:

		pagination_response
		Contains information about the pagination used.

		sites
		Contains information about the sites.
	"""

	input_map = {
		'PaginationResponse': ('pagination_response', PaginationResponse),
		'Sites': ['sites', Site],
		}

	output_map = {
		'pagination_response': 'PaginationResponse',
		'sites': 'Sites',
		}

	pagination_response = None
	sites = None

File: swagger/Models/GetSitesResponse.php
<?php

namespace App\Mindbody\Models;

/**
 * no description available
 */
class GetSitesResponse extends BaseModel
{
	/**
	 * Contains information about the pagination used.
	 * 
	 * @var PaginationResponse 
	 */
	public $paginationResponse;
	/**
	 * Contains information about the sites.
	 * 
	 * @var Site[] 
	 */
	public $sites;

	protected function getInputMap()
	{
		return [
			'PaginationResponse' => ['paginationResponse', PaginationResponse::class, null],
			'Sites' => ['sites', 'array', Site::class],
			];
	}

	protected function getOutputMap()
	{
		return [
			'paginationResponse' => ['PaginationResponse', PaginationResponse::class, null],
			'sites' => ['Sites', 'array', Site::class],
			];
	}
}
